src/views/Posts/four-oh-four.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import { mdx } from 'mdx.macro';
+
+const Content = mdx`
```

Comment:
I'm guessing the 404 page isn't in the scope of this PR but if this is the main 404 page (and not only for posts) I'd put a more generic message.

העמוד אינו קיים

Reply:
Yeah this was just a placeholder for me